pkg/options/linux.go

```diff
@@ -0,0 +1,12 @@
+package options
+
+import "github.com/bugsnag/bugsnag-cli/pkg/utils"
+
+type LinuxOptions struct {
+	Path          utils.Paths `arg:"" name:"path" help:"The path to the directory or file to upload" type:"path" default:"."`
+	ApplicationId string      `help:"A unique application ID, usually the package name, of the application"`
+	ProjectRoot   string      `help:"The path to strip from the beginning of source file names referenced in stacktraces on the BugSnag dashboard" type:"path" default:"."`
+	VersionCode   string      `help:"The version code of this build of the application"`
```

Comment:
Although it's in the API, we can effectively leave it unused as it's an Android concept. So let's delete this option from the CLI.